pkg/azuredisk/azuredisk.go

```diff
@@ -90,11 +93,14 @@ type hostUtil interface {
 // DriverCore contains fields common to both the V1 and V2 driver, and implements all interfaces of CSI drivers
 type DriverCore struct {
 	csicommon.CSIDriver
-	ready        chan struct{}
-	deviceHelper optimization.Interface
-	nodeInfo     *optimization.NodeInfo
-	ioHandler    azureutils.IOHandler
-	hostUtil     hostUtil
+	ready                                    chan struct{}
+	deviceHelper                             optimization.Interface
+	nodeInfo                                 *optimization.NodeInfo
+	ioHandler                                azureutils.IOHandler
+	hostUtil                                 hostUtil
+	enableAzureClientAttachDetachRateLimiter bool
```

Comment:
Please do not add any configuration related fields to `DriverCore`. The V2 driver uses the `AzDiskDriverConfiguration` directly making these values redundant. We don't build the V1 driver out of this branch, so you can either add the fields to the `Driver` struct or simply pass default values for the V1 driver.